app/models/court_application_defendant_summary.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+class CourtApplicationDefendantSummary
+  include ActiveModel::Model
+
+  attr_reader :defendant
+  attr_accessor :defendant_id
+
+  def initialize(attributes = {})
+    super
+    @defendant ||= Defendant.find(attributes[:defendant_id])
```

Comment:
this can trigger an exeption,
is the find always called with existent `defendant_id` ?

if so no problem,
otherwise maybe better to use `find_by`

Reply:
Doesn't fail